Add flag support to syslog logging method

benrubson opened the pull request:
Hello,

This PR solves [req #60525](https://bz.apache.org/bugzilla/show_bug.cgi?id=60525).
We can now specify `ErrorLog:[facility]:[tag]`.

Very useful when there are several Apache instances on the same server.

Thank you 👍 

Ben

Commit:
Add flag support to syslog logging method

Commit:
Add flag support to syslog logging method, doc

jimjag commented:
Thanks for the patch. This patch is specific to httpd 2.4.x and changes to that branch requires that the code/functionality 1st live in trunk before backporting to 2.4. I will look into what is required to add this to trunk

benrubson commented:
Thank you very much @jimjag 👍 
The code is slightly different in trunk, `openlog()` is no more in `log.c` but in `mod_syslog.c`.
However, should be quite easy to adapt :)
Thank you again !

jimjag commented:
This has been ported to trunk and will propose for backport for 2.4.x

Thx again!

benrubson commented:
Should I leave this opened until backported ?
Thx again too 👍

benrubson commented:
Jim, when do you think you will be able to backport this for 2.4.26 release ?
Many thx 👍

benrubson commented:
Hi @jimjag, some news regarding the 2.4.x backport ?
Many thanks 👍

jimjag commented:
Will likely not be in 2.4.26 but in 2.4.27, to allow for backport testing

benrubson commented:
OK Jim.
2.4.26 is almost ready to be released ?
As this PR brings a small feature, I would have thought it would have been quite easy & fast to test it :)
But I may be missing some points :)
Thank you 👍

benrubson commented:
@jimjag what about proposing the patch in 2.4.x STATUS file, as 2.4.26 has now been released ? :)
(I'm really expecting this in 2.4)
Thank you very much again for your support 👍

Commit:
Add flag support to syslog logging method, streamline

based on https://github.com/apache/httpd/commit/cbf210e4ada463b0a736b150915ab5c2a03e6a85

benrubson commented:
I just reworked this 2.4.x patch based on @jimjag port to trunk : https://github.com/apache/httpd/commit/cbf210e4ada463b0a736b150915ab5c2a03e6a85
Thank you 👍 
Ben

jimjag commented:
Proposed for back port... full patch at http://home.apache.org/~jim/patches/syslog-tag.patch

benrubson commented:
Jim, thank you very much 👍
Really glad :)
Thank you again 👍 👍 👍

benrubson commented:
Just committed by Yann : https://github.com/apache/httpd/commit/8b0f0d0945de073ed67c4af837f31c02d580097a 👍 
Thank you all !
